packages/rn-tester/js/examples/Layout/LayoutAnimationExample.js

```diff
@@ -342,14 +428,30 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
   },
+  choiceContainer: {
+    flexDirection: 'row',
+    marginBottom: 10,
+  },
+  optionStyle: {
+    padding: 6,
+    marginHorizontal: 5,
+    borderColor: '#ccc',
+    borderWidth: 1,
+    borderRadius: 5,
+  },
+  optionHeader: {
+    fontWeight: '500',
+    marginBottom: 5,
+  },
 });
 
 exports.title = 'Layout Animation';
-exports.documentationURL = 'https://reactnative.dev/docs/layoutanimation';
 exports.description = 'Layout animation';
```

Comment:
Could we have a short description about the component here, so that it looks consistent in the API listing page as well?

Reply:
![image](https://user-images.githubusercontent.com/14032427/90687028-d4fd7a00-e289-11ea-893e-9efc1c5910dc.png)

Reply:
Done!